test/automation/src/sql/connectionDialog.ts

```diff
@@ -30,8 +31,10 @@ export class ConnectionDialog extends Dialog {
 
 	private static readonly CONNECT_BUTTON_SELECTOR = '.modal .modal-footer a[aria-label="Connect"]:not(.disabled)';
 	async connect(): Promise<void> {
+		await this.notificationToast.closeNotificationToasts();
```

Comment:
Do we know what notification toasts were blocking the input? This is fine for now but in general we should try to make sure that notification toasts are handled by the thing that displays them.

Reply:
![VSCode_Smoke_Tests__Electron__VSCode_Smoke_Tests__Electron__Query_Editor_can_new_file__connect_and_execute (2)](https://user-images.githubusercontent.com/23462877/116305062-1c60d200-a758-11eb-80dc-4e6b1f4a801f.png)

Reply:
Before a test suite/case is run, can we close any outstanding toasts/dialogs? Just have a cleanup utility method or something?

Reply:
Yes I can add that. But I think we still want to check for toasts right before clicking a wizard footer button since a toast could still pop up after a test suite has started.

Reply:
My only concern is by doing so we may be hiding actual issues - for example error popups or something.

In this case I just filed a bug that the SQL Proj extension should provide a setting to disable these popups from happening. 

https://github.com/microsoft/azuredatastudio/issues/15247

I'm not completely opposed to the idea of closing the toasts before each test, but I feel like right now that isn't really actually necessary if they just make the popup behave like it generally should.

Reply:
Sounds reasonable @Charles-Gagnon. I'd be curious if we could restrict this to exclude error notifications (but this presumes that there's a different CSS class associated with error notifications)

Reply:
That's an interesting thought - I'd feel better about that for sure. Something to investigate. 

(in general I'd still rather our default be to not try to close toasts and instead make sure that the functionality that shows them allows for disabling when appropriate. Trying to "guess" when popups are going to appear is going to be error prone anyways - if we close all toasts what's stopping another one from appearing between that and when we try to actually click the button?)